services/app-api/forms/wp.json

```diff
@@ -1127,6 +1127,212 @@
+            "addEditModalAddTitle": "Add funding source and projected expenditures",
+            "addEditModalEditTitle": "Edit funding source and projected expenditures",
+            "tableHeader": "Initiative name <br/> Work Plan topic",
+            "addEditModalHint": "Provide projected quarterly expenditures, by funding source, for this initiative. Actual quarterly expenditures will be reported in the recipient’s Semi-Annual Report (SAR).",
+            "emptyDashboardText": "Empty",
+            "dashboardTitle": "Funding Sources"
+          },
+          "modalForm": {
+            "id": "tb-modal",
+            "fields": [
+              {
+                "id": "initiative_wpTopic",
+                "type": "radio",
+                "validation": "radio",
+                "props": {
+                  "label": "Funding source:",
+                  "hint": "Enter a dollar amount using a $ or a percentage with %. Enter 0 for quarters with no projected expenditures. Enter N/A for quarters you do not expect to report.",
+                  "choices": [
+                    {
+                      "id": "2VLpZ9A92OivbZhKvY8pE4hB65c",
+                      "label": "State-equivalent funds"
+                    },
+                    {
+                      "id": "2VLpZ9A92OivbZhKvY8pE4",
+                      "label": "Capacity-building funds"
+                    },
+                    {
+                      "id": "2VLpZCNtbcjRPq3evd1NI6",
+                      "label": "Administrative cooperative agreement funding"
+                    },
+                    {
+                      "id": "2VLpZEkkfhqqS95l7LFj0y",
+                      "label": "Self-direction(*if applicable)"
+                    },
+                    {
+                      "id": "2VLpZDJ9qaKKOk78ztBdiB",
+                      "label": "MFP funding for qualified HCBS"
+                    },
+                    {
+                      "id": "2VLpZCRWieGr1Z49QX5Aqc",
+                      "label": "MFP funding for demonstrated services"
+                    },
+                    {
+                      "id": "2VLq8ASzZNfxbu520if449",
+                      "label": "MFP funding for supplemental services"
+                    },
+                    {
+                      "id": "2VLq8ASzZNfxbu520if449",
+                      "label": "Other, specify",
+                      "children": [
+                        {
+                          "id": "initiative_wp_otherTopic",
+                          "type": "textarea",
+                          "validation": {
+                            "type": "text",
+                            "nested": true,
+                            "parentFieldName": "initiative_wpTopic"
+                          }
+                        }
+                      ]
+                    }
+                  ]
+                }
+              },
+              {
+                "id": "2VLqDuo2xxhCWuYuQdZTGmEnuYv",
```

Comment:
all of these fields can have named ID's since they aren't choices in a choice list. So fundingSources_q32023 would work here and a similar scheme for the others